src/doc/src/guide/creating-a-new-workspace.md

```diff
@@ -0,0 +1,75 @@
+# Creating a New Workspace
+
+A [workspace][def-workspace] is a collection of one or more packages, called workspace members, that are managed together.
+
+In this chapter we implement the creation of a workspace `new_workspace` from scratch, in which two members `foo-bin` and `bar-lib` are included.
+
+As mentioned in [[workspace] section](../reference/workspaces.md#the-workspace-section), the workspace must have at least one member, either the root package or a virtual manifest. Here we recommend setting the root of the workspace to be a virtual manifest. The reason is as follows:
+
+- The crate namespace at the Cargo level is flat. The tree layout creates another hierarchy and increases the possibility of inconsistencies. 
+- With a flat structure, adding or splitting crates is very easy. 
+- Even larger flattened lists are easier to see at a glance and easier to maintain than smaller tree structures.
+
+Let's look at how a workspace containing a virtual inventory should be created.
+
+Currently `Cargo` does not provide automatic creation of workspaces using the command line. You need to create the `Cargo.toml` file under the folder `new_workspace`:
+
+```toml
+# [new_workspace]/Cargo.toml
+[workspace]
+```
+
+If using a virtual workspace, then the version of [resolver] needs to be specified in the table (if not, the default version of resolver for a workspace is `1`, even if the default resolver version for workspace members is `2`), for example:
+
+```toml
+# [new_workspace]/Cargo.toml
+[workspace]
+resolver = "2"
+```
+
+You can then proceed to create workspace members. As mentioned above, if you have a lot of workspace members, we recommend designing the workspace with a flat structure. So add all the workspace members in the directory `new_workspace/crates`.
+
+There are several ways to add members to the workspace, we recommend using the command `cargo new/init`. For example, add the members `foo-bin` and `bar-lib` to the workspace:
+
+```console
+$ cargo new crates/foo-bin
+$ cargo new crates/bar-lib
+```
+
+Cargo will automatically add members to `Cargo.toml`:
+
+```toml
+# [new_workspace]/Cargo.toml
+[workspace]
+resolver = "2"
+members = [ "bar-lib","foo-bin"]
+
+[package]
+# ...
+```
+
+The project at this point contains the following files:
+
+```console
+$ cd new_workspace
+$ tree .
+.
+├── Cargo.toml
+└── crates
+    ├── bar-lib
+    │   ├── Cargo.toml
+    │   └── src
+    │       └── main.rs
+    └── foo-bin
+        ├── Cargo.toml
+        └── src
+            └── main.rs
+
+5 directories, 5 files
+```
+
+Up to this point, we have achieved creating a workspace that contains multiple `crate` members through a step-by-step process.
```

Comment:
I don't understand why quoting "`crate`". Also, could put some basic usages of workspace here maybe?

```suggestion
Up to this point, we have a workspace with two members.
Whenever you run `cargo build` under the workspace root directory, Cargo builds all member at once.
Instead of building the entire workspace, you could use the `--package`/`-p` flag to select certain packages.
For example, `cargo build -p foo-bin` will build only `foo-bin` package.
```